stable/prometheus/Chart.yaml

```diff
@@ -1,10 +1,12 @@
 name: prometheus
-version: 2.0.4
-description: A Prometheus Helm chart for Kubernetes. Prometheus is a monitoring system and time series database.
+version: 3.0.0
+description: Prometheus is a monitoring system and time series database.
 home: https://prometheus.io/
+icon: https://github.com/prometheus/prometheus.github.io/blob/master/assets/prometheus_logo-cb55bb5c346.png
```

Comment:
For this to work in Monocular it needs the raw link, e.g. https://raw.githubusercontent.com/prometheus/prometheus.github.io/master/assets/prometheus_logo-cb55bb5c346.png